contrib/OtlpGrpc/Exporter.php

```diff
@@ -0,0 +1,136 @@
+<?php
+
+declare(strict_types=1);
+namespace OpenTelemetry\Contrib\OtlpGrpc;
+require __DIR__ . '/../../vendor/autoload.php';
```

Comment:
Is there a reason to require the autoload here? I think it should stay only in the entry-point of an application.